src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -230,7 +416,18 @@ public static EntityBreakBlockEvent createEntityBreakBlock(Game game, Cause caus
         return createEvent(EntityBreakBlockEvent.class, values);
     }
 
-    public static EntityChangeBlockEvent createEntityChangeBlock(Game game, Cause cause, Entity entity, BlockLoc block, BlockSnapshot replacementBlock) {
+    /**
+     * Creates a new {@link EntityChangeBlockEvent}.
+     *
+     * @param game The game instance for this {@link GameEvent}
+     * @param cause The cause of the event, can be null
+     * @param entity The entity involved in this event
+     * @param block The block affected by this event
+     * @param replacementBlock The block that will replace the existing block
+     * @return A new instance of the event
+     */
+    public static EntityChangeBlockEvent
+            createEntityChangeBlock(Game game, Cause cause, Entity entity, BlockLoc block, BlockSnapshot replacementBlock) {
```

Comment:
The [Google style guide](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.5.1-line-wrapping-where-to-break) states:
> There is no comprehensive, deterministic formula showing exactly how to line-wrap in every situation. Very often there are several valid ways to line-wrap the same piece of code.

So either way of wrapping is allowed in this situation. I'll wrap the arguments to be consistent however IMO it's neater to move the method name down.